### DISCUSSION TITLE
软件后台闪退

### DISCUSSION THREAD

**Discussion by ghost** [Planshit/Tai#250] [General]
源
Tai

摘要
已停止工作

日期
‎2023/‎8/‎11 9:48

状态
已发送报告

描述
错误的应用程序路径:	E:\Tai\Tai.exe

问题签名
问题事件名称:	APPCRASH
应用程序名:	Tai.exe
应用程序版本:	1.5.0.1
应用程序时间戳:	95ef1bed
故障模块名称:	UIAutomationCore.dll
故障模块版本:	7.2.18362.693
故障模块时间戳:	d708c103
异常代码:	c0000005
异常偏移:	000e07ca
OS 版本:	10.0.18363.2.0.0.256.161
区域设置 ID:	2052
其他信息 1:	2beb
其他信息 2:	2beba6fb4680d73a8c78ca7c24ccdb46
其他信息 3:	9e66
其他信息 4:	9e66d0b07564bd5b934c1974771457cc

关于该问题的额外信息
存储段 ID:	ae2ad2300607b48f623336bbbe813144 (1311452096477868356)

**Comment by noberumotto** [comment]
感谢提供了关键的信息
